### DISCUSSION TITLE
Laravel 11

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#896] [Q&A (Help)]
What about migration to LV 11 for BKP6.x ?

**Comment by pxpm** [comment]
What about it @blondie63 ?

Any help needed? Anything you spotted it's not working ? 

Cheers

**Reply by blondie63** [reply]
Honestly, I haven't tried it, I was waiting for your official communications regarding Backpack 6 and various Addons

**Reply by pxpm** [reply]
I think we didn't sent any email blast about it, since there is nothing special to note about it. It will probably be written on the next progress report 👍 

If you find any issue let us know, but it should be just changing `laravel/framework` to 11 and make sure you are not locking package versions, for example locking `backpack/crud` to 6.6 (since the L11 support was added in 6.7). 

Cheers

**Reply by blondie63** [reply]
ok i'll try with some my projects and let you know

regards

Mauro

**Comment by jcastroa87** [comment]
Thanks, @blondie63. Is there is anything you need help feel free to re-open this issue.

Cheers.
